### ISSUE TITLE
Feature request: allocation confirmations

### ISSUE THREAD

**Issue by philipbelesky** [czlee/tabbycat#252]
Just putting this out there for some quick feedback. WADL is looking for a feature whereby adjudicators can 'confirm' their shifts. Their availability is collected at the start of the year, but they want people to indicate they have seen that they have been allocated to a round and that they are *still* available for it. My thoughts:

- I don't want to spend too much time on this feature. It's pretty niche, and may be superseded by a more complete rego system later. 
- My first thought is to use a system similar to public feedback/ballots by adjs - ie adjs get a unique URL that takes them to an availability page 
- This page shows all their current allocations across all active tournaments. These are unset radio buttons - available or unavailable 
- My first thought is that these correspond to a true/false/null on the DebateAdjudicator object. This doesn't have to have any affect on the draw or anything functional
- Admins either see this information on a dedicated page or as an extra column on the normal draw pages. 

As a side note there are now quite a few instances of multi tournament views. I'll probably work in a less hackish system for managing these soon.

**Comment by czlee** [comment]
Yeah, I wonder to what extent this falls into my "would be more appropriate in a separate system" category. Also, psychologically, would a form with checkboxes actually solve the problem, as in, would it be taken seriously enough by adjudicators? What is the root problem that WADL is trying to solve? For example, do adjudicators actually need to be e-mailed instead for this to be practical?

**Comment by czlee** [comment]
(That said, feel free to hack away, within reasonable bounds, if it needs to be done before we have time to think everything through.)

**Comment by philipbelesky** [comment]
The root problem is essentially that people's have schedule/availability changes between the start of the year (when things are collected in advance) and when the draw is done. Apparently they've found that nagging over facebook/text is much more effective than emailing, hence why the unique URL idea should fit fine. 

I've mentioned other functionality that could be offered by a more dedicated rego system, but they are unlikely to use it (school admins are apparently to stuck in their ways / not technologically adept) and are relatively confident this feature is the only thing they need on this kind of front. If there was the chance that more scheduling related stuff would need to be done I'd probably split this off, but as is I think it can be tacked on in the manner described above.

**Comment by czlee** [comment]
When do they send out this link? Are they planning to pester people over Facebook/text, and demand confirmation each time they send it? How long do adjudicators have to reply? I don't know what the culture of the league is like but if the problem is adjudicators needing to be reminded to show up or say something if they can't, I feel like there's a lot more to that than a page where adjudicators can indicate whether they're still available. In fact, if the problem is that adjudicators need to actually respond, I wonder if they'd also be too lazy/busy to open another web page, unique as it might be. Also, if it's a unique URL for each adj, you can't mail merge that in Facebook/text.

**Comment by philipbelesky** [comment]
So adjs are allocated 2-3 weeks prior to the round, the link goes out soon after, results are tallied about a week later. Problems thereafter are resolved via facebook nagging. They only have ~25 adjudicators across all levels per year, so no real need for mail merge and the like.
